fsleyes-plugin-shimming-toolbox/fsleyes_plugin_shimming_toolbox/tabs/mask_tab.py

```diff
@@ -229,3 +234,40 @@ def create_sizer_sphere(self):
         )
         sizer = self.run_component_sphere.sizer
         return sizer
+    
+    def create_sizer_bet(self):
+        path_output = os.path.join(__CURR_DIR__, "output_mask_bet")
+        input_text_box_metadata = [
+            {
+                "button_label": "Input",
+                "button_function": "select_from_overlay",
+                "name": "input",
+                "required": True
+            },
+            {
+                "button_label": "fractional intensity threshold",
+                "name": "f_param",
+                "default_text": "0.5",
+                "required": True
+            },
+            {
+                "button_label": "Vertical gradient",
+                "name": "g_param",
+                "default_text": "0.0",
+                "required": True
+            },
+            {
+                "button_label": "Output File",
+                "button_function": "select_folder",
+                "default_text": os.path.join(path_output, "anat.nii.gz"),
```

Comment:
```suggestion
                "default_text": os.path.join(path_output, "bet.nii.gz"),
```

Reply:
Maybe

Reply:
no because this is the image use to create the mask (bet is weird)

Reply:
Isn't it the anatomical masked?